### DISCUSSION TITLE
Reverse hindsight

### DISCUSSION THREAD

**Discussion by ap0ll0nas** [gopro/labs#221] [Feature Requests]
It would be great to have an option that lets the camera continuously record and once the record button is pressed, the clip is only the last x seconds, where x is user defined. The use case for this is that often an activity may start when the remote is out of range or when a user doesn't have a remote or when the user doesn't want to focus on using the remote. The user can then take his time to start and do the activity and only worry about the recording aspect after the activity is done. Typically this will be a repetitive action, i.e. there will be a well-defined time interval from the end of the activity back to when it started - with x allowing for some extra few secs, just in case.

This is kind of a reverse hindsight but not exactly, I set the title like that for convenience.

**Comment by dnewman-gpsw** [comment]
How does this differ from duration record?

**Reply by ap0ll0nas** [reply]
I could not find the definition of duration record. My understanding is that this performs a recording of x sec from the moment you press the record button onwards. I am looking backwards, keep the last x sec prior to the moment of pressing the record button - that stops the recording and initiates a new one.

**Comment by Tjay311** [comment]
I would love to see this function too. Currently, when HINDSIGHT is enabled, there are only 2 options to choose from. Either **15** seconds or **30** seconds of reverse capture. I would like to be able to reverse capture **60** seconds when using hindsight.
